moveit_ros/robot_interaction/src/robot_interaction.cpp

```diff
@@ -329,15 +329,14 @@ void RobotInteraction::decideActiveEndEffectors(const std::string& group, Intera
     }
   }
 
-  for (std::size_t i = 0; i < active_eef_.size(); ++i)
+  for (EndEffectorInteraction& eef : active_eef_)
   {
     // if we have a separate group for the eef, we compute the scale based on it;
     // otherwise, we use the size of the parent_link
-    active_eef_[i].size = active_eef_[i].eef_group == active_eef_[i].parent_group ?
-                              computeLinkMarkerSize(active_eef_[i].parent_link) :
-                              computeGroupMarkerSize(active_eef_[i].eef_group);
-    ROS_DEBUG_NAMED("robot_interaction", "Found active end-effector '%s', of scale %lf",
-                    active_eef_[i].eef_group.c_str(), active_eef_[i].size);
+    eef.size = eef.eef_group == eef.parent_group ? computeLinkMarkerSize(eef.parent_link) :
```

Comment:
```suggestion
    eef.size = (eef.eef_group == eef.parent_group) ? computeLinkMarkerSize(eef.parent_link) :
```
Unrelated, but we might as well make this more readable.

Reply:
Agree, corrected.